src/store/app/actions.js

```diff
@@ -1,31 +1,37 @@
 import { AppState } from 'react-native';
 import { createAction } from 'redux-actions';
 import * as ROUTES from '../../constants/routes';
-import { reset } from '../../store/navigation/actions';
+import { reset } from '../navigation/actions';
 import { initAuth } from '../auth/actions';
-import { APP_STATE_CHANGE } from './types';
+import { APP_STATE_CHANGE, SET_PENDING } from './types';
+
 
 const appStateChangeAction = createAction(APP_STATE_CHANGE);
+const setPendingAction = createAction(SET_PENDING);
+
 
-export const appStateChange = (state) => (dispatch) => {
+export const appStateChange = state => (dispatch) => {
   dispatch(appStateChangeAction(state));
-}
+};
+
+export const setPending = bool => (dispatch) => {

```

Comment:
i think **bool** is **bad** name for function param, **isPending** would be better